webpack.config.js

```diff
@@ -11,6 +13,9 @@ module.exports = {
     new webpack.ProvidePlugin({
       $: "jquery",
       jQuery: "jquery"
+    }),
+    new MiniCssExtractPlugin({
```

Comment:
Is this to generate the "widget.css" file within the resources folder? Since you moved the CSS to be included within the React app?

Reply:
+1 on the jupyter stylings - I think the .less files used by jupyter are here: https://github.com/jupyter/notebook/tree/master/notebook/static/notebook/less
In particular, this one seems of interest: https://github.com/jupyter/notebook/blob/master/notebook/static/notebook/less/renderedhtml.less